src/types/common.d.ts

```diff
@@ -1,45 +0,0 @@
-/** Contexts common interface */
-export interface RenderContext {
```

Comment:
I think we'll need both `.d.ts` and `.ts`, no? The `.d.ts` should only have interfaces with no code.

I would suggest leaving this file as `src/types/common.d.ts` and adding a `src/common.ts` with just the assignment (for the glyph props.)

Reply:
I really do not know. I tried the declarations on tests with .d.ts and I did not get it to work. Then @wassertim showed that it work with .ts. If we have .d.ts and .ts they have to be in diferent directories otherwise #import does not tell the difference.
We also decided to put the exports on the normal files, I mean we did not do .d.ts for element with the interfaces like @wassertim did in his first migration. We had a discussion about that and you said that you prefered them self contained.

Reply:
It is your choice, I just wanted to higlight that it is inconsistent with the others.

Reply:
Thinking more about that I find it inconvenient to have them two imports one from `./common` and one from `./types/common` Then perhaps `types/common.d.ts` with interfaces and `flow.ts`where we start porting little by little out of `table.ts`?

Reply:
forget about it `common.ts` is fine